diff --git a/lib/dependency-manager-adapters/npm.js b/lib/dependency-manager-adapters/npm.js
index b4b7a691636215974c18f49756df0cd51837209f..10754c8a8a4e0fd77b5a59a4853ec2af0d90311c 100644
--- a/lib/dependency-manager-adapters/npm.js
+++ b/lib/dependency-manager-adapters/npm.js
@@ -124,16 +124,7 @@ module.exports = CoreObject.extend({
         throw new Error('buildManagerOptions must return an array of options');
       }
     } else {
-      if (this.useYarnCommand) {
-        if (mgrOptions.indexOf('--no-lockfile') === -1) {
-          mgrOptions = mgrOptions.concat(['--no-lockfile']);
-        }
-        // npm warns on incompatible engines
-        // yarn errors, not a good experience
-        if (mgrOptions.indexOf('--ignore-engines') === -1) {
-          mgrOptions = mgrOptions.concat(['--ignore-engines']);
-        }
-      } else if (mgrOptions.indexOf('--no-shrinkwrap') === -1) {
+      if (!this.useYarnCommand && mgrOptions.indexOf('--no-shrinkwrap') === -1) {
         mgrOptions = mgrOptions.concat(['--no-shrinkwrap']);
       }
     }
